src/main/scala/diplomaticobjectmodel/logicaltree/RocketComponentLogicalTrees.scala

```diff
@@ -0,0 +1,87 @@
+package diplomaticobjectmodel.logicaltree
```

Comment:
None of these component have anything to do with "Rocket" the core pipeline, so I would rename this file to DeviceLogicalTrees.scala

Reply:
fixed

Reply:
What is "fixed" about this? The file still isn't renamed.